.ci/jenkins/Jenkinsfile.prod.nightly

```
@@ -91,7 +91,12 @@ pipeline {
                         def mavenRepositoryFileUrl = "${env.STAGING_SERVER_URL}/openshift-serverless-logic/openshift-serverless-logic-${PRODUCT_VERSION}.nightly/maven-repository-${PME_BUILD_VARIABLES['datetimeSuffix']}.zip"
                         def topic = "VirtualTopic.qe.ci.ba.openshift-serverless-logic.${env.UMB_VERSION}.nightly.trigger"
                         def eventType = "openshift-serverless-logic-${env.UMB_VERSION}-nightly-qe-trigger"
-                        def messageBody = getMessageBody(mavenRepositoryFileUrl, env.ALREADY_BUILT_PROJECTS)
+                        def messageBody = getMessageBody(
+                            mavenRepositoryFileUrl, 
+                            env.ALREADY_BUILT_PROJECTS,
+                            ['serverlesslogic': PME_BUILD_VARIABLES['productVersion'], 'serverlesslogic-rhba': PME_BUILD_VARIABLES['productVersion']], 
```

Other suggestion
```suggestion
                            ['serverlesslogic': PME_BUILD_VARIABLES['productVersion'], 'rhba-serverlesslogic': PME_BUILD_VARIABLES['productVersion']], 
```
That will help for regex